tekton-catalog/pipeline-loops/examples/simplepipelineloop.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: simpletask
+spec:
+  params:
+    - name: word
+      type: string
+    - name: suffix
+      type: string
+  steps:
+    - name: echo
+      image: ubuntu
+      script: |
+        #!/usr/bin/env bash
+        echo "$(params.word)$(params.suffix)"
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: demo-pipeline
+spec:
+  params:
+  - name: word
+    default: "word"
+  - name: suffix
+    default: "suffix"
+  tasks:
+  - name: simpletask
+    taskRef:
+      name: simpletask
```

Comment:
Can you use pipeline.taskRef to refer to custom task? Also, does pipeline.taskSpec support custom task?
Since custom task is using the alpha api, it's better to put it under the pipeline.taskSpec if possible to avoid the Tekton client version conflict.

Reply:
Yes, use pipeline.taskRef is the only case of using custom task.